Python 2.7 build fails, no checksum available

Trying to build Qt on a mac OS system (13.7.6) with this command using the 5.15.16 branch:

```
./Build-qt.sh -j 50 -m ~/slicer/cmake/cmake-3.31.0-macos-universal/CMake.app/Contents/bin/cmake -d 13.0 -s  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk
```

Several dependencies build but then it seems Python 2.7 is needed, but [only 3.5 and later are available](https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/blob/master/CMakeLists.txt#L182-L325).

```
+ cd python-cmake-buildsystem-build
+ /Users/pieper/slicer/cmake/cmake-3.31.0-macos-universal/CMake.app/Contents/bin/cmake -DCMAKE_BUILD_TYPE:STRING=Release -DCMAKE_INSTALL_PREFIX:PATH=/Users/pieper/slicer/qt-easy-build/qt-everywhere-deps-5.15.16/python-cmake-buildsystem-install -DPYTHON_VERSION:STRING=2.7.15 -DBUILD_LIBPYTHON_SHARED:BOOL=ON -DENABLE_SSL:BOOL=OFF ../python-cmake-buildsystem
-- Checking which MACOSX_DEPLOYMENT_TARGET to use
-- Checking which MACOSX_DEPLOYMENT_TARGET to use - 13.3
-- The C compiler identification is AppleClang 14.0.3.14030022
-- The ASM compiler identification is AppleClang
-- Found assembler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Downloading https://www.python.org/ftp/python/2.7.15/Python-2.7.15.tgz
CMake Error at CMakeLists.txt:337 (message):
  Selected PY_VERSION [2.7.15] is not associated with any checksum.  Consider
  updating this CMakeLists.txt setting _download_2.7.15_md5 variable


-- Configuring incomplete, errors occurred!
```

--- a/Build-qt.sh
+++ b/Build-qt.sh
@@ -361,6 +361,8 @@ if [[ ! -d python-cmake-buildsystem ]]
 then
   git clone https://github.com/python-cmake-buildsystem/python-cmake-buildsystem.git
 fi
+# Checkout an older python-cmake-buildsystem commit that is compatible with Python 2.7
+(cd python-cmake-buildsystem; git checkout 39d95fd6f6bc78f4d7fa4217093281aa736517c9)
 cd python-cmake-buildsystem-build
 $cmake \
   -DCMAKE_BUILD_TYPE:STRING=Release \

